public-ui/src/app/views/quick-search/quick-search.component.css

```diff
@@ -20,10 +20,15 @@
 }
 
 .card-text.bottom {
-    padding-bottom: 1rem;
+    padding-bottom: .2rem;
 }
 
 .bolder {
     font-weight: bold;
     font-size: 16px;
+}
+
+span {
```

Comment:
This is awful generic css ... Do we really want to apply these rules to every span on this page?  Not real friendly going forward if have to add some spans to this page. Maybe make a specific class  for spans than need this ?

Reply:
Gave it a class name.